### DISCUSSION TITLE
Alarm Control Panel Stuck in Entry-Delay / Pending

### DISCUSSION THREAD

**Discussion by stephensalex** [tsightler/ring-mqtt#774] [Support Q&A] [answered]
Hi all,

I seem to be experiencing an issue where my Alarm Control Panel device is stuck in the state of **Pending** due to the **AlarmState** perpetually being reported as **entry-delay**. I have tried arming and disarming the alarm multiple times to clear the status with no luck. Note that the status is showing correctly in the Ring application. I have also reconfigured the keypad in the Ring application and rebooted the base station and Home Assistant, again to no avail. This issue has occurred previously but somehow resolved itself, this doesn't seem to be the case on this occasion.

Looking at the MQTT messages, the device in Home Assistant is indeed reflecting the MQTT messages correctly.

I am currently driving my heating schedule automations based on whether our house is occupied or not (using the Alarm Control Panel status as a proxy for this). I would really appreciate if you have any suggestions for resolving this recurring issue?

Thanks in advance,
Alex
![MQTT Debug 2](https://github.com/tsightler/ring-mqtt/assets/3968898/beca09e1-a21f-4634-931a-db87e108db01)
![Pending Alarm](https://github.com/tsightler/ring-mqtt/assets/3968898/c6c81af8-0987-451e-a45a-c84c5571ef65)
![Alarm Info](https://github.com/tsightler/ring-mqtt/assets/3968898/503e4f34-7baa-4f59-99ea-bf7618038bbc)
![MQTT Debug 1](https://github.com/tsightler/ring-mqtt/assets/3968898/b82b4f43-34e3-433e-a28f-689d7e18c5b8)

**Comment by tsightler** [comment, marked as answer]
Did you try restarting the addon (not restarting HA, the specifically the addon)?

**Reply by stephensalex** [reply]
Ah I assumed that restarting HA would achieve the same result. Yes restarting only the add on seems to have cleared the problem, can't believe it was such a simple fix. Thanks so much.

**Comment by stephensalex** [comment]
Resolved
